src/app/navbar/navbar.component.html

```diff
@@ -11,16 +11,15 @@ <h1 class="h4 logo">
          *ngIf="authService.isAuthenticated()"
          dropdown
          [(isOpen)]="isSettingsDropdownOpen">
+      <a class="button button-link" href="//{{ apiHost }}">API</a>
```

Comment:
Not idea. These links are inside the `<div class="dropdown">` which is semantically incorrect. Moving them outside destroys the existing styling, and we're not quite using the bootstrap navbar.

Reply:
this should go to `//{{apiHost}}/accounts/api/`